docs/reference/services.rst

```diff
@@ -4,10 +4,64 @@
 MicroOVN services
 =================
 
+MicroOVN has lots of its functionality separated into distinct services that can
+be controlled via MicroOVN enable/disable or snap/systemd, only the former is
```

Comment:
exemplify the commands, i.e. `controlled via ``microovn enable`` and ``microovn disable`` commands.

Drop mentioning snap/systemd and the comment after the comma.